x-pack/plugins/security/server/routes/authorization/roles/model/elasticsearch_role.ts

```diff
@@ -83,13 +83,13 @@ function transformRoleApplicationsToKibanaPrivileges(
     };
   }
 
-  // if space privilege assigned globally, we can't transform these
+  // if there is a reserved privilege assigned to the non-wildcard application, we won't transform these
```

Comment:
__question:__ would you mind explaining somewhere in the comment within this function what these wildcard and non-wildcard applications are?